src/psyclone/tests/domain/lfric/lfric_scalar_mdata_test.py

```diff
@@ -234,6 +254,21 @@ def test_ad_int_scalar_type_no_sum():
             in str(excinfo.value))
 
 
+def test_ad_logical_scalar_type_no_sum():
```

Comment:
(I think) you could parameterise the integer test:

    @pytest.mark.parameterize("scalar_type", ["gh_integer", "gh_logical"]])

and then you don't need this second, separate test.

Reply:
Melikes this very much, thanks for the suggestion! 😃